cedar-policy-cli/src/lib.rs

```diff
@@ -95,17 +95,42 @@ pub enum Commands {
     Link(LinkArgs),
     /// Format a policy set
     Format(FormatArgs),
+    /// Translate natural policy syntax to JSON (except comments)
+    TranslatePolicy(TranslatePolicyArgs),
     /// Translate JSON schema to natural schema syntax and vice versa (except comments)
     TranslateSchema(TranslateSchemaArgs),
+    /// Visualize a set of JSON entities to the graphviz format.
+    /// Warning: Entity visualization is best-effort and not well tested.
+    Visualize(VisualizeArgs),
     /// Create a Cedar project
     New(NewArgs),
+    /// Partially evaluate an authorization request
+    PartiallyAuthorize(PartiallyAuthorizeArgs),
+}
+
+#[derive(Args, Debug)]
+pub struct TranslatePolicyArgs {
+    /// The direction of translation,
+    #[arg(long)]
+    pub direction: PolicyTranslationDirection,
+    /// Filename to read the policies from.
+    /// If not provided, will default to reading stdin.
+    #[arg(short = 'p', long = "policies", value_name = "FILE")]
+    pub input_file: Option<String>,
+}
+
+/// The direction of translation
+#[derive(Debug, Clone, Copy, ValueEnum)]
+pub enum PolicyTranslationDirection {
+    /// Human policy syntax -> JSON
+    HumanToJson,
```

Comment:
Do we want to avoid introducing this flag as `human-to-json` in 3.3 only to rename it to `cedar-to-json` in 4.0?

Reply:
Fair question. I think it's better to leave as `human-to-json` here for consistency with the schema translation arguments. The change in 4.0 will then affect both translation commands.